spec/functions/ensure_php_packages_spec.rb

```diff
@@ -0,0 +1,11 @@
+require "spec_helper"
+
+describe "ensure_php_packages" do
+  it "creates a resource in the catalogue" do
```

Comment:
You could add a test case to cover arguments[2]

Reply:
will do